src/components/Combobox/Combobox.tsx

```diff
@@ -79,6 +81,9 @@ export type ComboboxProps<T> = {
 
   /** The maximum height (in pixels) of the MultiCombobox dropdown. Defaults to 300. */
   maxHeight?: number;
+
+  /** Whether a clear selection control should be available to the user */
+  withClearSelectionControl?: boolean;
```

Comment:
Nit: `clearSelection` is better IMHO, as `withXXX` is mostly used for HOCs

Reply:
hmmm.. `clearSelection` looks like a function though. I wouldn't expect it to be a `boolean`. Also if I changed it to `enableClearSelection` it would be conceptually wrong, since user can **always** clear the sselection by pressing `ESC`. 

That's why I have used the `xxxControl` suffix, to be clear I'm referring to an element and not a functionality. How about `showClearSelectionControl`, `enableClearSelectionControl`, `displayClearSelectionControl`?

Reply:
I would prefer the `clearSelection` approach but that's just me. It's definitely not a blocker.